tests/test_wave_01.py

```diff
@@ -1,8 +1,7 @@
 from app.models.task import Task
 import pytest
 
-
-@pytest.mark.skip(reason="No way to test this feature yet")
+pytest.mark.skip(reason="No way to test this feature yet")
```

Comment:
This works, because removing the `@` makes it not a decorator, but I recommend commenting it out rather than removing the @ symbol.
```suggestion
#@pytest.mark.skip(reason="No way to test this feature yet")
```